core/modules/user/user.module

```diff
@@ -59,7 +59,7 @@ function user_theme() {
   return array(
     'user_picture' => array(
       'variables' => array('account' => NULL),
-      'template' => 'templates/user-picture',
+      'template' => 'templates/user-profile',
```

Comment:
At first I took out this line, but something went wrong so I pointed at the user-profile tpl